### PR TITLE
Application stories sf

### PR DESCRIPTION
**Pull request by StephanieFriend**
- Added class to application.css to move current application title to the right
- Added application.all to the pets controller for more access
- Create logic in the favorites index page to display the names of the applied for pets
- Adjusted application_process_spec test to correlate with story 18
- Create a passing test for story 18
- Test coverage at 97%

### CODE REVIEW

**Comment by StephanieFriend** [comment]
Resolved merge conflicts in seed file

File: app/controllers/pets_controller.rb [hunk 1]
```diff
@@ -35,6 +35,7 @@ def destroy
   end
 
   def favorites
+    @applications = Application.all
```
Comment (tylertomlinson) [inline app/controllers/pets_controller.rb:38]: Nice!

File: app/views/pets/favorites.html.erb [hunk 2]
```diff
@@ -1,7 +1,14 @@
 <div class="form-group">
 
-  <h1>Favorite Pets</h1>
+  <h1>Favorite Pets<span class="applications-header">Current Applications</span></h1>
 
+  <div class="applications-header">
```
Comment (tylertomlinson) [inline app/views/pets/favorites.html.erb:5]: Looks good

File: spec/features/applications/can_see_pet_names_on_favorites_page_spec.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
```
Comment (tylertomlinson) [inline spec/features/applications/can_see_pet_names_on_favorites_page_spec.rb:1]: Can probably just included this test on the favorites show page test. Will make note to refactor

**Comment by tylertomlinson** [comment]
Nice work!

**Comment by tylertomlinson** [comment]
Whoops

**Pull request merged**